src/pages/iou/SplitBillDetailsPage.js

```diff
@@ -64,10 +64,10 @@ function getReportID(route) {
 
 function SplitBillDetailsPage(props) {
     const reportAction = props.reportActions[`${props.route.params.reportActionID.toString()}`];
-    const personalDetails = OptionsListUtils.getPersonalDetailsForLogins(reportAction.originalMessage.participants, props.personalDetails);
+    const personalDetails = OptionsListUtils.getPersonalDetailsForAccountIDs(reportAction.originalMessage.participantAccountIDs, props.personalDetails);
```

Comment:
This caused a [regression](https://github.com/Expensify/App/issues/21044), because we changed the shape of Onyx data without creating a migration to go along with it.